src/scene/mesh.js

```diff
@@ -723,7 +723,7 @@ class Mesh extends RefCountedObject {
             } else {
                 // destination data is array
                 indices.length = 0;
-                indices.push(streamIndices);
+                indices.push(...streamIndices);
```

Comment:
Nice, but lets avoid using spread operator here on these potentially larger arrays due to https://github.com/nodejs/node/issues/16870
(We've had an experience in other code with this)

Reply:
Looks like a good-old for-loop would handles both cases equally here (untested assumption)

Reply:
yep

Reply:
@chrissterling - would you have time to do this change please?